modules/notification/indexer/indexer.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2018 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package indexer
+
+import (
+	"code.gitea.io/git"
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/notification"
+)
+
+type indexerReceiver struct {
+}
+
+var (
+	receiver notification.NotifyReceiver = &indexerReceiver{}
+)
+
+func ini() {
```

Comment:
Think this should be `init` here as well

Reply:
done.